opentelemetry-user-events-metrics/src/exporter/mod.rs

```diff
@@ -24,7 +24,10 @@ impl MetricsExporter {
         // This is unsafe because if the code is used in a shared object,
         // the event MUST be unregistered before the shared object unloads.
         unsafe {
-            let _result = tracepoint::register(trace_point.as_ref());
+            let result = tracepoint::register(trace_point.as_ref());
```

Comment:
nit - Looking at the [implementation ](https://github.com/microsoft/LinuxTracepoints-Rust/blob/e4c927d0a5f9e915058716ebcd6d0434d5376a86/eventheader/src/native.rs#L472)of this [method](https://github.com/microsoft/LinuxTracepoints-Rust/blob/e4c927d0a5f9e915058716ebcd6d0434d5376a86/eventheader/src/native.rs#L422), it seems to returns [errno](https://man7.org/linux/man-pages/man3/errno.3.html), which means it can returns negative values other than -1? Do we need to handle that: e.g to print the errno value if it is negative/failure ?